scripts/update-notice.py

```diff
@@ -63,8 +63,7 @@
                     "version": "1.2-4ubuntu0.20.04.1~esm1",
                     "is_source": "false",
                     "source_link": (
-                        "https://launchpad.net"
-                        "/ubuntu/+source/libxmltok"
+                        "https://launchpad.net" "/ubuntu/+source/libxmltok"
```

Comment:
Can you change "https://launchpad.net/" "ubuntu/+source/libxmltok" to https://launchpad.net/ubuntu/+source/libxmlto?